client/src/main/java/org/apache/iotdb/client/AbstractClient.java

```diff
@@ -488,19 +495,27 @@ private static void printBlockLine(boolean printTimestamp, int colCount,
         blockLine.append(StringUtils.repeat('-', maxValueLengthForShow[i - 1])).append("+");
       }
     } else {
-      int tmp = Integer.MIN_VALUE;
-      for (int i = 1; i <= colCount; i++) {
-        int len = resultSetMetaData.getColumnLabel(i).length();
-        tmp = tmp > len ? tmp : len;
-      }
-      maxValueLength = tmp;
       if (printTimestamp) {
         blockLine.append("+").append(StringUtils.repeat('-', maxTimeLength)).append("+");
       } else {
         blockLine.append("+");
       }
-      for (int i = 0; i < colCount - 1; i++) {
-        blockLine.append(StringUtils.repeat('-', maxValueLength)).append("+");
+      if (resultSetMetaData.getColumnName(2).equals(GROUPBY_DEVICE_COLUMN_NAME)) {
+        maxValueLength = sensorColumnLength;
```

Comment:
As you can see in the TLexer.g, I make 'DEVICE' a keyword. Therefore, you can not create a timeseries called "root.xxx.Device" because 'Device' is now reserved as a keyword and is never recognized as a general `Identifier`, while timeseries is only made up of `root` and `Identifier`.
```
Identifier
    :
    (Letter | '_') (Letter | Digit | '_' | MINUS)*
    ;
```
```
timeseries
  : KW_ROOT (DOT identifier)+
  -> ^(TOK_PATH ^(TOK_ROOT identifier+))
  ;
```
For example, I tried the following sql:
```
IoTDB> create timeseries root.vehicle.d0.DEVICE with datatype=INT32,encoding=RLE
Msg: Meet error while parsing SQL: Statement format is not right: parsing error,statement: create timeseries root.vehicle.d0.DEVICE with datatype=INT32,encoding=RLE
It costs 0.006s
```

Can you accept this constraint, i.e., 'device' is now a reserved word ?

Reply:
I think it is fine.